zulipterminal/helper.py

```diff
@@ -187,14 +184,14 @@ def set_count(id_list: List[int], controller: Any, new_count: int) -> None:
     assert new_count == 1 or new_count == -1
     messages = controller.model.index['messages']
     unread_counts = controller.model.unread_counts  # type: UnreadCounts
-
-    _set_count_in_model(id_list, new_count, messages, unread_counts)
+    message_list = [messages[id] for id in messages]
```

Comment:
`...for id in id_list` ?

Could `message_list` be better named?

Reply:
Thanks for catching this!